microcosm_postgres/encryption/v2/column.py

```diff
@@ -180,8 +180,13 @@ def _prop_comparator(cls) -> Comparator[T] | InstrumentedAttribute:
         )
 
     def encrypted(self) -> Mapped[bytes | None]:
+        info = {
```

Comment:
:nit should we use a TypedDict? Might help people understand what goes into the info obj?